README.rst

```diff
@@ -25,13 +25,16 @@ Development
   make
 
 
-Create a tunnel to localhost using something like https://ngrok.com/download::
+If your sentry server is hosted behind a firewall without publich IP, you should
```

Comment:
I'm not sure if I'd recommend using ngrok for any production environment. It's a developer tool.